src/pages/DataAccessRequestApplication_new.js

```diff
@@ -221,763 +205,235 @@ class DataAccessRequestApplicationNew extends Component {
+                setExternalCollaboratorsCompleted,
               }))
             ]),
 
             div({className: 'step-container'}, [
+              h(DarValidationMessages, {
+                validationMessages: validationMessages.darErrors,
+                showValidationMessages
+              }),
+
               h(DataAccessRequest, {
-                formData: this.state.formData,
-                formFieldChange: this.formFieldChange,
-                setCollaborationLetter: this.setCollaborationLetter,
-                setIrbDocument: this.setIrbDocument,
+                formData: formData,
+                datasets,
+                dataUseTranslations,
+                formFieldChange,
+                batchFormFieldChange,
+                uploadedCollaborationLetter,
+                updateCollaborationLetter,
+                uploadedIrbDocument,
+                updateUploadedIrbDocument: updateIrbDocument,
+                setDatasets,
               })
             ]),
 
             div({className: 'step-container'}, [
+              h(DarValidationMessages, {
+                validationMessages: validationMessages.rusErrors,
+                showValidationMessages
+              }),
+
               h(ResearchPurposeStatement, {
-                darCode: darCode,
-                formFieldChange: this.formFieldChange,
-                formData: this.state.formData,
+                darCode: formData.darCode,
+                formFieldChange: formFieldChange,
+                formData: formData,
               })
             ]),
 
             div({className: 'step-container'}, [
-              h(DataUseAgreements, {
-                darCode: darCode,
-                problemSavingRequest,
-                attestAndSave: this.attestAndSave,
-                ConfirmationDialogComponent,
-                partialSave: this.partialSave,
-                prevPage: this.prevPage,
-                step1Invalid,
-                step2Invalid,
-                step3Invalid,
-                libraryCardInvalid,
-                showValidationMessages,
-                updateShowValidationMessages: this.updateShowValidationMessages,
-                goToStep: this.goToStep
-              })
-            ])
-          ])
+              // just here to mock save/submit
+              div({ className: 'flex flex-row', style: { justifyContent: 'flex-end' }, }, [
+                a({
+                  id: 'btn_save', isRendered: isNil(formData.darCode), onClick: () => {},
+                  className: 'button button-white',
+                  style: {
+                    marginRight: '2rem',
+                  }
+                }, ['Save']),
+                a({
+                  id: 'btn_submit', isRendered: isNil(formData.darCode), onClick: () => mockSubmit(),
+                  className: 'button button-blue '
+                }, ['Attest and Send']),
+              ])
+            ]),
```

Comment:
these are here just in place of the new DataUseAgreements panel. clicking save does nothing, clicking submit runs the validation code.